.travis.yml

```diff
@@ -33,7 +32,7 @@ before_script:
   - if [[ $TRAVIS_PHP_VERSION != hhvm ]] ; then cat ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini|grep redis; fi;
   - if [[ $TRAVIS_PHP_VERSION != hhvm ]]; then phpenv config-rm xdebug.ini; fi;
   - phpenv rehash
-  - composer install --no-interaction --prefer-dist --dev
+  - composer install --no-interaction --prefer-dist --dev --ignore-platform-reqs
```

Comment:
it's required for running tests on hhvm (they are not ready yet, but when they will be then tests will try to go again there).